### PR TITLE
Model

### PR DESCRIPTION
**Pull request by szympie**

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import pytest
+from brats import models
+
+import os
+import torch
+
+import pytest
+import nibabel as nib
+from brats.data import datasets
+import numpy as np
```
Comment (khrynczenko) [inline tests/test_models.py]: A lot of unused imports.

File: brats/models.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    def forward(self, input):
+        out = self.conv1(input)
+        out = self.conv2(out)
+        return out
+
+
+class UNet3D(nn.Module):
+    def __init__(self, in_channels=3, out_channels=1, init_features=32):
+        super(UNet3D, self).__init__()
```
Comment (khrynczenko) [inline brats/models.py]: `super().__init__()`

File: tests/test_models.py (hunk 1 above)
Comment (szympie) [reply tests/test_models.py]: Fixed

File: brats/models.py (hunk 2 above)
Comment (szympie) [reply brats/models.py]: Fixed

**Pull request merged**